src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -150,6 +150,35 @@
  * <pre>
  * &lt;module name="JavadocMethod"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * &#47;**
+ *  * A sample function to add two numbers
+ *  * @param number1 The first number
+ *  * @param number2 The second number
+ *  * @return An integer
+ *  *&#47;
+ * public int addTwoNumbers(int number1, int number2){   // ok, param tags present
+ *      return number1 + number2;    // ok, return tag present
+ * }
```

Comment:
Try to keep `// ok` and `// violation` comments parallel.
```
public int addTwoNumbers(int number1, int number2){   // ok, param tags present
return number1 + number2;                                                   // ok, return tag present
}
```
If you face line length error, modify variable names, modify the example to tackle it.